core/src/tests/bareos_test_sockets.cc

```diff
@@ -105,8 +118,16 @@ static int create_listening_server_socket(int port)
     perror("setsockopt");
     return -5;
   }
-
-  if (listen(listen_file_descriptor, 3) < 0) {
+  int listenresult = -1;
+  for (int i = 0; i < 6; i++) {
+    listenresult = listen(listen_file_descriptor, 3);
+    if (listenresult == 0) {
+      break;
```

Comment:
Same here, would it make sense to log in case it was retired and successful, how many tries were needed?